### PR TITLE
editoast: isolate a few tests (naming rolling stocks)

### PR DESCRIPTION
**Pull request by shenriotpro**
Part of https://github.com/osrd-project/osrd/issues/5441

### CODE REVIEW

**Comment by hamz2a** [comment]
LGTM

File: editoast/src/models/rolling_stock/mod.rs [hunk 1]
```diff
@@ -251,35 +251,35 @@ pub mod tests {
     use super::RollingStockModel;
     use actix_web::web::Data;
 
-    pub fn get_fast_rolling_stock() -> RollingStockModel {
```
Comment (leovalais) [inline editoast/src/models/rolling_stock/mod.rs:254]: You could move this function in fixtures.rs and call it from `named_fast_rolling_stock`

File: editoast/src/models/rolling_stock/mod.rs [hunk 2]
```diff
@@ -251,35 +251,35 @@ pub mod tests {
...
+        let mut rs: RollingStockModel =
+            serde_json::from_str(include_str!("../../tests/example_rolling_stock_1.json"))
+                .expect("Unable to parse");
+        rs.name = Some(name.to_string());
+        rs
     }
 
-    pub fn get_other_rolling_stock() -> RollingStockModel {
-        serde_json::from_str(include_str!(
+    pub fn get_other_rolling_stock(name: &str) -> RollingStockModel {
```
Comment (leovalais) [inline editoast/src/models/rolling_stock/mod.rs]: Likewise

**Review by leovalais — APPROVED** [review]
LGTM, a suggestion:

**Review by hamz2a — APPROVED** [review]
LQTM

**Pull request merged**